### PR TITLE
clarify that persistent congestion is a contigous loss across all PN spaces

### LINKED ISSUE #3939: It's unclear if persistent congestion is a per-PN-space property
My understanding is that it is *not*, but it's unclear.

### PR DESCRIPTION
**Pull request by kazuho**
Fixes #3939.

### CODE REVIEW

**Comment by kazuho** [comment]
@marten-seemann It sounds like that your complaint is not against the change in this PR by itself. Reading your comments on #3939 it is my understanding that you think the change to the text is fine.

I'd assume that your complaint is rather about the pseudo-code not clarifying how QUIC stacks can implement `InPersistentCongestion`.

Regarding how, I could well be wrong, but I think it is not that difficult to implement InPersistentCongestion (see https://github.com/quicwg/base-drafts/pull/3889#discussion_r459160599).

I am positive to adding a description regarding how QUIC stacks might implement InPersistentCongestion, though I am not sure if we can come up with a pseudo-code that explains the algorithm linked above, considering the fact that we do not describe how an endpoint might implement a sentmap. So maybe adding a few lines of comments to the pseudo-code would be a good thing to do.

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -826,16 +826,17 @@ packets.
...
-threshold is to enable a sender to use initial PTOs for aggressive probing, as
-TCP does with Tail Loss Probe (TLP; see {{RACK}}), before establishing
-persistent congestion, as TCP does with a Retransmission Timeout (RTO; see
-{{?RFC5681}}). The RECOMMENDED value for kPersistentCongestionThreshold is 3,
-which is approximately equivalent to two TLPs before an RTO in TCP.
+sent over a long enough period of time across all the packet number spaces, the
+network is considered to be experiencing persistent congestion.  Commonly, this
+can be established by consecutive PTOs, but since the PTO timer is reset when a
+new ack-eliciting packet is sent, an explicit duration must be used to account
+for those cases where PTOs do not occur or are substantially delayed. The
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:833]: This business about establishing PC using PTO is really confusing and I would remove this sentence.

```suggestion
network is considered to be experiencing persistent congestion.  The
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -826,16 +826,17 @@ packets.
...
+rationale for this threshold is to enable a sender to use initial PTOs for
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md:834]:
```suggestion
rationale for this threshold is to allow a sender to initially use PTOs for
```

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (kazuho) [reply draft-ietf-quic-recovery.md]: I'm not the editor, but I think I agree. The rationale for choosing 3 as the multiplier is explained below. That's sufficient.

**Comment by janaiyengar** [comment]
#3961 subsumes this.

**Pull request closed without merging**